go/base/static/css/conversations.less

```diff
@@ -1,3 +1,6 @@
+@import "./lesshat";
```

Comment:
Not sure about this. We just use it for [.user-select(none);](.user-select(none);), which just stops people from selecting the diagram area, right? I'm not sure how it helps.